codegen/core/src/main/java/software/amazon/smithy/python/codegen/ClientGenerator.java

```diff
@@ -484,6 +495,11 @@ async def _handle_attempt(
             writer.write("""
                             # Step 7i: sign the request
                             if auth_option and signer:
+                                logger.debug("HTTP request to sign: %s", context.transport_request)
+                                logger.debug(
+                                    "Signer properties: %s",
+                                    auth_option.signer_properties
+                                )
```

Comment:
I can build this to check but if you have a quick answer, what does the output of these look like? I haven't checked our repr's for them.

Reply:
The `signer_properties` value would print like a normal python dictionary.

`auth_option` is either `HTTPAuthOption | None` and at this point we've checked `auth_option` isn't None. I've copied over the `HTTPAuthOption` class below for convenience:
```py
@dataclass(kw_only=True)
class HTTPAuthOption:
    """Auth scheme used for signing and identity resolution."""

    # The ID of the scheme to use. This string matches the one returned by
    # HttpAuthScheme.scheme_id
    scheme_id: str

    # Parameters to pass to IdentityResolver.get_identity.
    identity_properties: dict[str, Any]

    # Parameters to pass to HttpSigner.sign.
    signer_properties: dict[str, Any]
```